sdk/monitor/monitor-opentelemetry/src/metrics/quickpulse/liveMetrics.ts

```diff
@@ -675,6 +675,9 @@ export class LiveMetrics {
   }
 
   private parseDocumentFilterConfiguration(response: PublishResponse | IsSubscribedResponse): void {
+    if (!response.documentStreams) {
+      return;
+    }
     response.documentStreams.forEach((documentStreamInfo) => {
```

Comment:
nit, these (may) still throw if the `response.documentStreams` is not of the "expected" type which happens to have a `forEach` function. Likewise if response is null / undefined

Reply:
Updated to a more robust check. It now determines if `documentStreams.forEach` is a function as well.